modules/BiomesAPI/build.gradle

```diff
@@ -35,17 +25,25 @@ buildscript {
     }
 }
 
-// Handle some logic related to where what is
-sourceSets {
-    main.java.outputDir = new File("$buildDir/classes")
-    test.java.outputDir = new File("$buildDir/testClasses")
-}
-JavaPluginConvention convention = project.getConvention().getPlugin(JavaPluginConvention.class);
-SourceSet mainSourceSet = convention.getSourceSets().getByName("main");
+import groovy.json.JsonSlurper
+import org.reflections.Reflections
+import org.reflections.util.FilterBuilder
+import org.reflections.scanners.SubTypesScanner
+import org.reflections.scanners.TypeAnnotationsScanner
+import org.reflections.util.ConfigurationBuilder
 
 ext {
     // Read environment variables, including variables passed by jenkins continuous integration server
     env = System.getenv()
+
+    // This is a fun one ... when versions switched to dynamic -SNAPSHOT or not based on branch existing modules using `master` would suddenly try publishing releases
+    // This won't work without additionally doing constant version bumps (perhaps via Git tags) - but too much work to switch around all modules at once
+    // Complicating things more the use of publish.gradle to centralize logic means modules and engine bits are treated the same, yet we need to vary modules
+    // Temporary workaround: default modules to bypass release management: master branch builds will still make snapshot builds for the snapshot repo
+    // If a module actually wants release management simply include `"isReleaseManaged" : true` in module.txt - this is needed for the engine repo embedded modules
+    // One option would be to slowly convert modulespace to default to a `develop` + `master` setup living in harmony with associated automation/github tweaks
+    // Alternatively one more round of refactoring could switch to Git tags, a single `master` branch, and possible other things to help match snaps/PR builds somehow?
```

Comment:
Heavyly in favor of git tags, as most modules don't have complicated branching going on anyways. But you're right, should not be done in this PR, and needs some proper thoughtwork to figure out details and edge cases...